snap/info_test.go

```diff
@@ -59,13 +59,15 @@ func (s *infoSuite) TestSideInfoOverrides(c *C) {
 		EditedDescription: "fixed desc",
 		Revision:          snap.R(1),
 		SnapID:            "snapidsnapidsnapidsnapidsnapidsn",
+		DeveloperID:       "deviddeviddeviddeviddeviddevidde",
```

Comment:
Is this a random string or a readable developer username?  The copy & paste from the line above might give an improper feeling.